tests/testUtils.js

```diff
@@ -68,10 +68,10 @@ const files = [
   {path: inputConstantsFilePath, content: inputConstantsFileContent}
 ]
 
-exports.createTestEndpoint = (callback) => {
+exports.createTestEndpoint = callback => {
   if (!fs.existsSync(pathToDirectory)) {
     fs.mkdir(pathToDirectory, err => {
-      if (err) return reject(err)
+      if (err) throw Error(err)
```

Comment:
shouldn't we be passing the error to the callback function, and then return that function.

Reply:
You are correct, this should be passed to the callback as an error and let the handler of the error deal with throwing it. Legacy code i havent updated :(

Reply:
done in 6f8796b